app/controllers/catalog_controller.rb

```diff
@@ -8,6 +8,24 @@ class CatalogController < ApplicationController
   include Arclight::FieldConfigHelpers
   include Ngao::ComponentMetadataHelper
 
+  # OVERRIDE v1.4.0 Arclight::Catalog#hierarchy to force large row count for expand_all
+  def hierarchy
+    @expand_all = params[:expand_all] == 'true'
+
+    # search_params_logic_context = {
+    #   expand_all: @expand_all
+    # }
```

Comment:
Was this method left in on purpose?

Reply:
thanks for the catch, i'll remove it before merging